crates/swc_css_parser/src/parser/value/mod.rs

```diff
@@ -1326,6 +1330,214 @@ where
     }
 }
 
+// <urange> =
```

Comment:
I love this comment

Reply:
In future I will add more these comments, it allow to read code without looking on strange logic inside :star: